scripts/src/models/all.ts

```diff
@@ -25,86 +24,86 @@ if (dbConfig.options.storage && !/^[./]/.test(dbConfig.options.storage)) {
 const sequelize = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.passive, dbConfig.options);
 const models = {
 	_items: [] as GenericModel[],
-	push: function<T extends GenericModel>(item: T): T {
+	push<T extends GenericModel>(item: T): T {
 		this._items.push(item);
 		return item;
 	},
-	sync: function(): Promise<GenericModel[]> {
+	sync(): Promise<GenericModel[]> {
 		return Promise.all<GenericModel>(this._items.map(item => {
 			return item.sync().then(model => {
 				console.log(`table ${ model.name } created`);
 				return model;
 			});
 		}));
-	}
+	},
 };
 
 const User = models.push(sequelize.define("users", {
-	id: { type: Sequelize.STRING, primaryKey: true }, //internal id
+	activated_date: Sequelize.DATE,
 	app_id: Sequelize.STRING,
 	app_user_id: Sequelize.STRING,
 	created_date: Sequelize.DATE,
-	activated_date: Sequelize.DATE
-}));
+	id: { type: Sequelize.STRING, primaryKey: true }, // internal id
+}) as GenericModel);
```

Comment:
I condemn this.
:(

Reply:
yeah me too. but it's just to get this linter in.
as we've already agreed, this part is going to look completely different.